tools/RELEASE.md

```diff
@@ -36,9 +36,14 @@ Change to the latest librdkafka version in the following files:
                      make sure to strip the leading "v" from the version.
                      E.g., `0.11.6` rather than `v0.11.6`
 
-Commit this change:
+Change to the latest version of the confluent-librdkafka-plugins in:
 
-    $ git commit -m "librdkafka version bump to v0.11.6" .travis.yml .appveyor.yml
+ * `tools/install-interceptors.sh`
```

Comment:
```suggestion
 * `tools/install-interceptors.sh` - edit and change version
```

Reply:
Just above this we state the following: 

"Change to the latest version of the confluent-librdkafka-plugins in:"

To be clear this in addition to this statement not a replacement correct?

Reply:
I just wanted to clarify that the file needs editing rather than being run, since referencing a shell script in backticks usually suggests the latter.

Reply:
sure works for me just wanted to make sure